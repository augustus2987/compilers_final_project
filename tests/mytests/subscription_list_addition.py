l = {1: [1], 2: [2]}
l[1] = l[1] + [2, 3]
l[2] = [1] + l[2] + [3]
print l
print l[1]
print 7 + 3
print -----7
a = input()
b = input()
print input() +  -7 + 3 if a else (7 if b else input())
