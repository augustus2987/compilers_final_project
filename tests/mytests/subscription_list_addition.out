{1: [1, 2, 3], 2: [1, 2, 3]}
[1, 2, 3]
10
-7
0
